plugin/build.gradle.kts

```diff
@@ -124,11 +115,9 @@ license {
 
 publishing {
     publications {
-        register<MavenPublication>("plugin") {
+        create<MavenPublication>("pluginMaven") {
+            groupId = "org.gradle"
             artifactId = "test-retry-gradle-plugin"
-            artifact(shadowJar.get()) {
-                classifier = null
-            }
```

Comment:
@jbartok This should now be done automatically, right?

Reply:
Yes